analysis/Analysis/Section_3_4.lean

```diff
@@ -196,7 +196,10 @@ theorem SetTheory.Set.example_3_4_9 (F:Object) :
     F ∈ ({0,1}:Set) ^ ({4,7}:Set) ↔ F = object_of f_3_4_9_a
     ∨ F = object_of f_3_4_9_b ∨ F = object_of f_3_4_9_c ∨ F = object_of f_3_4_9_d := by sorry
 
-/-- Lemma 3.4.10.  One needs to provide a suitable definition of the power set here. -/
+/--
+  Lemma 3.4.10.  One needs to provide a suitable definition of the power set here.
+  See Exercise 3.4.6 (i) for a hint on how to construct it.
```

Comment:
I could also embed the hint directly here but it's maybe nicer not to spoil it right away? The book achieves that by placing the hint in the exercises after the section content.